### ISSUE TITLE
internal "locked warning" have to be themed

### ISSUE THREAD

**Issue by djay** [plone/Products.CMFPlone#715]
# user problem

The themers job should be as easy as possible which means once they include the toolbar they don't don't have to theme any other backend internal plone UI, only whats public content.
Currently this warning appears in the content area for only logged in users.
"Locked This item was locked by XXXXXX 1 minute ago.
If you are certain this user has abandoned the object, you may [unlock] the object. You will then be able to edit it."

# options
1. The toolbar shows a lock icon. The themer could just remove the warning from the theme.
2. Show the warning when the user clicks edit. 
  - maybe the edit should get the lock icon since thats the action thats prevented.
3. Show the warning in the history view since that has the lock icon

@plone/ux-team feedback?

**Comment by vangheem** [comment]
new toolbar will show the lock button fyi. Still shows in the content though.

**Comment by djay** [comment]
yeah the lock icon is great. But I think option 2 of moving the lock icon to edit and having the warning appear when you click on the locked edit is the way to go.

**Comment by davilima6** [comment]
+1 for moving lock information next to Edit icon, which would display a tooltip similar to those on Folders Contents navbar containing full message: editing user and lock duration time. And if user ignores the tooltip there's the warning prompt anyway.

**Comment by jcerjak** [comment]
Related issue on locking: #1056

**Comment by jensens** [comment]
is this still an issue?

**Comment by davisagli** [comment]
The lock message is just one example; there are other messages that indicate special content states, such as the one about a plone.app.iterate working copy. I'd suggest a message at the very top or bottom of the window for these to avoid needing to style it to fit inside the content area.

**Comment by jensens** [comment]
Closing as outdated (4 years old). About editor warning/info box theming in Plone 5. Plone 6 Classic UI and Volto have entirely different theming approaches.
If you think this issue is still relevant please reopen it. Thank you for the original report!
